config/config.go

```diff
@@ -221,6 +221,9 @@ type ConsensusParams struct {
 
 	// sum of estimated op cost must be less than this
 	LogicSigMaxCost uint64
+
+	// a precision for assets
+	SupportAssetsPrecision bool
```

Comment:
Instead of a `bool`, can we just call this `MaxAssetDecimals` and have it be a `uint8`/`uint16`/`uint32` whose value is 19 after the upgrade? We'll need to set a maximum value anyway, and that way there's only one new proto variable instead of two.